package.json

```diff
@@ -28,47 +28,47 @@
     "rebuildsass": "npm rebuild node-sass"
   },
   "dependencies": {
-    "@angular/common": "10.2.5",
-    "@angular/compiler": "10.2.5",
-    "@angular/core": "10.2.5",
-    "@angular/forms": "10.2.5",
+    "@angular/common": "13.3.11",
+    "@angular/compiler": "13.3.11",
+    "@angular/core": "13.3.11",
+    "@angular/forms": "13.3.11",
     "@angular/http": "*",
-    "@angular/platform-browser": "10.2.5",
-    "@angular/platform-browser-dynamic": "10.2.5",
-    "@angular/router": "10.2.5",
+    "@angular/platform-browser": "13.3.11",
+    "@angular/platform-browser-dynamic": "13.3.11",
+    "@angular/router": "13.3.11",
     "core-js": "^3.6.4",
     "jsoneditor": "^9.1.8",
     "rxjs": "*",
     "webpack": "^4.46.0",
-    "zone.js": "^0.11.3"
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.900.2",
-    "@angular-devkit/schematics": "^9.0.2",
-    "@angular/cli": "10.0.5",
-    "@angular/compiler-cli": "10.0.8",
-    "@angular/language-service": "10.0.8",
-    "@types/jasmine": "3.3.8",
+    "@angular-devkit/build-angular": "~13.3.9",
+    "@angular-devkit/schematics": "^13.3.9",
+    "@angular/cli": "13.3.9",
+    "@angular/compiler-cli": "13.3.11",
+    "@angular/language-service": "13.3.11",
+    "@types/jasmine": "~3.6.0",
     "@types/node": "^13.7.1",
-    "codelyzer": "~5.0.0",
+    "codelyzer": "^6.0.0",
     "cpr": "^3.0.0",
     "jasmine": "^3.3.0",
-    "jasmine-core": "^3.3.0",
-    "jasmine-spec-reporter": "^4.2.1",
-    "karma": "~4.1.0",
-    "karma-chrome-launcher": "~2.2.0",
+    "jasmine-core": "~3.6.0",
+    "jasmine-spec-reporter": "~5.0.0",
+    "karma": "~6.4.0",
+    "karma-chrome-launcher": "~3.1.0",
     "karma-cli": "~1.0.1",
     "karma-coverage-istanbul-reporter": "^2.0.0",
-    "karma-jasmine": "~2.0.1",
-    "karma-jasmine-html-reporter": "^1.4.0",
-    "ng-packagr": "^9.0.0",
-    "protractor": "~5.4.0",
+    "karma-jasmine": "~4.0.0",
+    "karma-jasmine-html-reporter": "^1.5.0",
+    "ng-packagr": "^13.3.1",
+    "protractor": "~7.0.0",
     "rimraf": "^2.7.1",
     "ts-node": "~7.0.0",
     "tsickle": "^0.29.0",
-    "tslib": "^1.9.0",
+    "tslib": "^2.0.0",
     "tslint": "~6.1.3",
```

Comment:
I guess tslint can be removed since Angular dropped support for it

Reply:
may be setup `@angular-eslint` instead?

Reply:
@hardikpatel043 what u think about it?

Reply:
@mariohmol  Sure i will try it out

Reply:
Migration is completed and PR is updated. I had to fix few lint issues. I haven't touched few lint fixes and disabled rule for those places as i don't want to break anything. Please share your feedback if you have any.